rmgpy/molecule/pathfinder.py

```diff
@@ -243,28 +243,49 @@ def compute_atom_distance(atom_indices, mol):
 
 def find_allyl_delocalization_paths(atom1):
     """
-    Find all the delocalization paths allyl to the radical center indicated
-    by `atom1`. Used to generate resonance isomers.
+    Find all the delocalization paths allyl to the radical center indicated by `atom1`.
     """
     cython.declare(paths=list, atom2=Atom, atom3=Atom, bond12=Bond, bond23=Bond)
     
     # No paths if atom1 is not a radical
     if atom1.radicalElectrons <= 0:
         return []
 
-    # Find all delocalization paths
     paths = []
     for atom2, bond12 in atom1.edges.items():
         # Vinyl bond must be capable of gaining an order
-        if (bond12.isSingle() or bond12.isDouble()) and atom1.radicalElectrons:
```

Comment:
Because it's being checked several lines above:
`    if atom1.radicalElectrons <= 0:   return []`
I don't mind keeping it here, perhaps for better code readability? Or perhaps keeping the condition here and removing the above `<= 0` check?

Reply:
Ah, I didn't notice that. This is fine.